README.md

```diff
@@ -21,3 +21,13 @@ After creating the account:
 - > `npm login` to login into NPM account
 - > `npm publish` to publish the plugin into NPM
 
+## To use on IOS
+- > `[[SentinelSDK alloc] initKoin:@"https://app-stage.sensys-iot.com"]` add into app delegate file
```

Comment:
This is a public repo, so lets not add any urls here please

Reply:
Done